pkgs/servers/monitoring/nagios/plugins/smartmon.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, smartmontools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "check_smartmon";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner  = "driehuis";
+    repo   = "Nagios_check_smartmon";
```

Comment:
I am not a fan of that tabulation, but I will ignore it. Otherwise, LGTM.

Reply:
Gotcha! I didn't intend to make those tabs but if my editor does give me a big red warning!

I'll have a tabs fixed for the next version update.

Reply:
"Tabulation" here is not exactly about `\t` whitespace, but about this "pretty formatting". It is mostly unneeded.